pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -324,10 +330,15 @@ private static void start() {
         if (cfg.getBoolean(ConfigFile.RESTRICT_REGISTER_ROUTE)) {
             whitelist(API.REGISTRATION, cfg.getStringList(ConfigFile.AUTH0_IP_WHITE_LIST));
         }
+        if (cfg.getBoolean(ConfigFile.RESTRICT_AUTH0_LOG_EVENT_ROUTE)) {
```

Comment:
Is the list of IPs different for the log events hook, or can we use the same ones that are in `ConfigFile.RESTRICT_REGISTER_ROUTE`?

Reply:
Yes, looks the same. Will fix it.